### PR TITLE
Rov 73 add the name field in the component html

### PR DESCRIPTION
**Pull request by Varuno8**

### CODE REVIEW

File: src/app/application/enum/country.enum.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// countries.enum.ts
```
Comment (rachittyagi) [inline src/app/application/enum/country.enum.ts:1]: this enum is not part of the work required for this card, please remove this and move it to its own branch

File: src/app/application/enum/enum.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// app.component.ts
```
Comment (rachittyagi) [inline src/app/application/enum/enum.component.ts:1]: this component is not needed

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: src/app/application/enum/enum.component.ts (hunk 2 above)
Comment (Varuno8) [reply src/app/application/enum/enum.component.ts]: deleted the folder

File: src/app/application/enum/country.enum.ts (hunk 1 above)
Comment (Varuno8) [reply src/app/application/enum/country.enum.ts]: deleted the folder

File: src/app/application/enum/enum.component.ts (hunk 2 above)
Comment (Varuno8) [reply src/app/application/enum/enum.component.ts]: sure, i will delete this file

**Review by rachittyagi — APPROVED** [review]

**Pull request merged**